packages/react/src/ReactElementValidator.js

```diff
@@ -228,7 +228,11 @@ function validatePropTypes(element) {
 
 var ReactElementValidator = {
   createElement: function(type, props, children) {
-    var validType = typeof type === 'string' || typeof type === 'function';
+    var validType =
+      typeof type === 'string' ||
+      typeof type === 'function' ||
+      typeof type === 'symbol' ||
```

Comment:
What happens when it's an unrecognised symbol? Do we throw later?

Reply:
Same question for number.

Reply:
Currently there's nothing implemented to throw later, since @sebmarkbage suggested leaving this space open for 
 fun things (https://github.com/facebook/react/pull/10783#discussion_r145575197)

Reply:
What does the user see though? We should at least warn because if it silently does nothing, then making it do something could be a breaking change (if someone starts relying on it by accident).

Reply:
So the behavior we want is to warn if it's not equal to `REACT_FRAGMENT_TYPE`?